testnets/synternettestnet/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "synternettestnet",
+  "assets": [
+    {
+      "description": "The native staking and governance token of the testnet version of Synternet chain.",
+      "denom_units": [
+        {
+          "denom": "uamber",
+          "exponent": 0
+        },
+        {
+          "denom": "AMBER",
+          "exponent": 6
+        }
+      ],
+      "base": "uamber",
+      "name": "Synternet",
+      "display": "AMBER",
+      "symbol": "AMBER",
+      "logo_URIs": {
```

Comment:
I won't require it here, but I recommend defining trace to mainnet SYNT (as type: testnet-mintage)